views/messages/chits.erb

```diff
@@ -0,0 +1,4 @@
+<p> Hey you! this is Chitter</p>
+<p> Have something to Chit? </p>
+<p> Chit it now! </p>
```

Comment:
I imagine you were writing this here to make sure that you could write a good test. Do you know how to make this page use data from the database?